### ISSUE TITLE
Save images display size, Light theme.

### ISSUE THREAD

**Issue by meetinghouse** [meetinghouse/cms#136]
Andy,
I notice that the program is saving and loading all images full size. My understanding is that this is what causes the pages to load slow. Bones saves the pages display size, so the load time is faster.

Example is http://preservationarts.corbettresearchgroup.com/american_legion_post_48_hollywood, the very first image thumbnail (screenshot thumbnail.)displays an estimated 225 x 150, but if you download that image, it is 1500 x 1140. 

Equivalent page, in Bones, http://www.preservationarts.net/index.php/american_legion_post_48_hollywood.html, thumbnail (screenshot bones.) 
![bones](https://cloud.githubusercontent.com/assets/5215893/8024374/f9153204-0cfe-11e5-8273-66917727e62d.jpg)
![thumbnail](https://cloud.githubusercontent.com/assets/5215893/8024375/ff2697aa-0cfe-11e5-8d90-9829fd81c865.jpg)

displays the same (roughly) but is actual size when downloaded, 300 x 228.
![thumbnail](https://cloud.githubusercontent.com/assets/5215893/8024366/b062314c-0cfe-11e5-9c13-6217da41114c.jpg)
![am1](https://cloud.githubusercontent.com/assets/5215893/8024367/bd3f5764-0cfe-11e5-8dee-8a47b1e5137f.JPG)

**Comment by jboz62** [comment]
@vivek-chaudhari Vivek, do you know how we can add this functionality to the image upload? Something like what WordPress does when you upload an image it resizes it with different sizes for thumbnail, small, medium, actual size. Is there a way to do this, maybe a module we can add? Looking around for ideas. Thanks, -John

**Comment by jboz62** [comment]
@vivek-chaudhari This might be what we need: http://www.phpgang.com/image-upload-and-resize-using-image-intervention-in-laravel_889.html

This looks interesting too: http://www.camroncade.com/dealing-with-image-uploads-laravel/

**Comment by vivek-chaudhari** [comment]
@jboz62 Yes I can do this. Even I had make use of this image intervention library for cropping purpose in laravel.

**Comment by jboz62** [comment]
@vivek-chaudhari That sounds good, Vivek, but we don’t want to crop, we need to resize. It would be the same image, just at different sizes, like WordPress does.

**Comment by jboz62** [comment]
@vivek-chaudhari Any idea how much time this might take to do?

**Comment by vivek-chaudhari** [comment]
@jboz62 But this doesn't mean that images put on editor will be resized automatically, we need to think about it how we can resize images which are directly put on content editor.
Using given solution images which will be uploaded using upload option can be resized and doing this one will take almost 6-8 hours.

**Comment by jboz62** [comment]
@vivek-chaudhari From what I'm reading in that documentation, is looks like you can set the size and it will resize when you upload the image. Here's the code snippet on that:

`Route::post('/upload', function(){

	if (Input::hasFile('avatar'))
	{
	    $file = Input::file('avatar');
	    $file->move('uploads', $file->getClientOriginalName());

	    $image = Image::make(sprintf('uploads/%s', $file->getClientOriginalName()))->resize(200, 200)->save();
	}
});`

Isn't that possible to do?

**Comment by jboz62** [comment]
@vivek-chaudhari More on that: http://image.intervention.io/api/resize

**Comment by vivek-chaudhari** [comment]
@jboz62 This is possible but I am talking about images which are we put on wysing content editor when we create a post or a page.

**Comment by vivek-chaudhari** [comment]
@jboz62 Also what could be possible image size, we need to make single image or multiple images of different sizes of single image?

**Comment by jboz62** [comment]
@vivek-chaudhari Vivek, we added this feature, right? If so, please close issue. Thank you.
